features/intl.yml

```diff
@@ -0,0 +1,107 @@
+name: Internationalization API
```

Comment:
Can you leave a comment up here linking to the discussion and maybe mentioning that there's a possible future TODO of composing a feature out smaller interface-specific features? Mostly I care about linking to your most recent comment.